### DISCUSSION TITLE
How do I create an item with no article?

### DISCUSSION THREAD

**Discussion by KVonGit** [ThePix/QuestJS#112] [Q&A] [answered]
I've looked and looked, and I can't find how to make an item that doesn't have a capitalized name print without "a", "an", "the" or "some".

I'm fairly certain I've done this before, but I can't find it anywhere.

**Comment by Woodhouse3D** [comment]
Have you tried setting the article to an empty string?

**Reply by KVonGit** [reply]
Hello.

I tried all these, but they didn't change anything (I didn't expect the last one to work, but figured it was worth a shot):
```js
  definiteArticle:'',
  indefiniteArticle:'',
  article:'',
```

**Reply by KVonGit** [reply]
By the way, I don't know JS very well, and I always appreciate feedback if I'm wrong or if there's simply a better way to handle things.

**Comment by KVonGit** [comment, marked as answer]
Ah, I figured out how to cheat:

```js
  pronouns:lang.pronouns.plural,
  properNoun:true,
```

**Reply by ThePix** [reply]
Have you seen this page?
https://github.com/ThePix/QuestJS/wiki/Using-Neutral-Language
